frontend/packages/pipelines-plugin/src/components/pipelines/pipeline-topology/TaskListNode.tsx

```diff
@@ -43,10 +44,13 @@ const TaskListNode: React.FC<TaskListNodeProps> = ({ element, unselectedText })
   const { height, width } = element.getBounds();
   const { clusterTaskList, namespaceTaskList, onNewTask, onRemoveTask } = element.getData();
 
-  const options = [
-    ...namespaceTaskList.map((task) => taskToOption(task, onNewTask)),
-    ...clusterTaskList.map((task) => taskToOption(task, onNewTask)),
-  ];
+  const options = _.sortBy(
+    [
+      ...namespaceTaskList.map((task) => taskToOption(task, onNewTask)),
+      ...clusterTaskList.map((task) => taskToOption(task, onNewTask)),
+    ],
+    (o) => o.label,
```

Comment:
You could simply use sort function instead of lodash :) 
```suggestion
  const options =  [
      ...namespaceTaskList.map((task) => taskToOption(task, onNewTask)),
      ...clusterTaskList.map((task) => taskToOption(task, onNewTask)),
    ].sort((a, b) => a.label.localeCompare(b.label))
```

Reply:
Either or. Neither is wrong.
If we want to impose such restrictions, then we should write lint rules for them. Add it to the nit pile.

Reply:
Definitely my preferred way, but it's effectively the same. If we were not so swamped, I'd have gone for this change.